daq/test_modules/ipaddr_module.py

```diff
@@ -38,14 +37,8 @@ def __init__(self, host, tmpdir, test_name, module_config):
             ('dhcp change test', self._dhcp_change_test),
             ('analyze results', self._analyze)
         ]
-        self._logger = logger.get_logger('ipaddr_%s' % self.host_name)
-        log_folder = os.path.join(self.tmpdir, 'nodes', self.host_name, 'tmp')
-        os.makedirs(log_folder)
-        log_path = os.path.join(log_folder, 'activate.log')
-        self._file_handler = logging.FileHandler(log_path)
-        formatter = logging.Formatter(_LOG_FORMAT)
-        self._file_handler.setFormatter(formatter)
-        self._logger.addHandler(self._file_handler)
+        log_file = os.path.join(self.tmpdir, 'nodes', self.host_name, 'tmp', 'activate.log')
```

Comment:
Why is this changed? There’s no need to have a separate custom logging handler

Reply:
I separated out the logging changes to #865 -- we can address/finalize that first, and then come back to this PR.